### PR TITLE
Update rubocop

### PR DESCRIPTION
**Pull request by ereslibre**
Supersedes: https://github.com/kubic-project/velum/pull/382

### CODE REVIEW

File: spec/features/bootstrap_cluster_feature_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 require "rails_helper"
 
 # rubocop:disable RSpec/AnyInstance
-feature "Bootstrap cluster feature" do
+describe "Bootstrap cluster feature" do
```
Comment (vitoravelino) [inline spec/features/bootstrap_cluster_feature_spec.rb]: Are we changing this?

File: spec/features/bootstrap_cluster_feature_spec.rb [hunk 2]
```diff
@@ -27,7 +27,7 @@
       allow(Orchestration).to receive(:run)
     end
 
-    scenario "A user sees warning modal when trying to bootstrap 2 nodes", js: true do
+    it "A user sees warning modal when trying to bootstrap 2 nodes", js: true do
```
Comment (vitoravelino) [inline spec/features/bootstrap_cluster_feature_spec.rb]: This also?

File: spec/features/bootstrap_cluster_feature_spec.rb (hunk 1 above)
Comment (ereslibre) [reply spec/features/bootstrap_cluster_feature_spec.rb]: Rubocop screams, yes.

File: spec/features/bootstrap_cluster_feature_spec.rb (hunk 2 above)
Comment (ereslibre) [reply spec/features/bootstrap_cluster_feature_spec.rb]: Yeah...

File: app/controllers/concerns/api.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```
Comment (mssola) [inline app/controllers/concerns/api.rb]: This is only valid on ruby versions higher or equal than 2.3. Thus, I'd recommend adding the following on the rubocop file and removing all the comments:

```yaml
Style/FrozenStringLiteralComment:
  Enabled: false
```

**Comment by ereslibre** [comment]
It's the default in upstream @vitoravelino: https://github.com/backus/rubocop-rspec/commit/7fe393a1936977f3416f3d27f04775c214a6329f

**Comment by ereslibre** [comment]
Removed all the comments and added the directive to the rubocop config.

**Comment by ereslibre** [comment]
Merging as I have addressed @mssola comments and @vitoravelino gave his approval.

**Pull request merged**